tools/scoring/tests/dimensions/entity_identification_test.py

```diff
@@ -31,23 +30,26 @@ def setUp(self):
     featureful_file_path = (
         'tests/samples/proposed/entity_identification_virtual_guid.yaml')
     self.highest_score_argument = TestHelper.prepare_dimension_argument(
-        dimension_type=COMPLEX,
+        dimension=EntityIdentification,
```

Comment:
this is a bit confusing?
dimension is either simple or complex?

Reply:
Yes, and now it is defined on the classes to avoid the overhead of maintaining separate lists.

Reply:
I see, 
since `dimension is either simple or complex by definition` should we give another name for `dimension` in`dimension=EntityIdentification`?

Reply:
I could change the argument to `dimension_category` but I'm not sure that would add clarity, e.g.: `dimension_category=EntityIdentification.category`. The way it stands, the retrieval of the `category` value is abstracted similarly to how it is in `parse_config.py`.